docs/usage/management-api-walkthrough/04_catalog.md

```diff
@@ -14,23 +14,19 @@ Content-Type: application/json
 {
   "@context": {
     "edc": "https://w3id.org/edc/v0.0.1/ns/",
-    "odrl": "http://www.w3.org/ns/odrl/2/"
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
   },
-  "@type": "edc:CatalogRequest",
+  "@type": "CatalogRequest",
   "counterPartyAddress": "https://provider-control.plane/api/v1/dsp",
+  "counterPartyId": "<PROVIDER-BPN>",
   "protocol": "dataspace-protocol-http",
   "querySpec": {
-    "@type": "edc:QuerySpec",
     "offset": 0,
     "limit": 50,
     "sortField": "https://purl.org/dc/terms/type",
     "sortOrder": "ASC",
     "filterExpression": [
       {
-        "@context": {
-          "edc": "https://w3id.org/edc/v0.0.1/ns/"
-        },
-        "@type": "edc:Criterion",
```

Comment:
Is this tested? In the past, `leftOperands` had to be given in expanded format (so it appears this was wrong in the docs before).

Reply:
do you mean the `edc:someProperty` ?

Reply:
yep, `https://w3id.org/edc/v0.0.1/ns/someProperty` I've seen work in the past, `edc:someProperty` (even with appropriate context present) not.

Reply:
Yes, it was already in the form of `edc:someProperty` but is not correct. Will fix it